src/wallet/mod.rs

```diff
@@ -1902,11 +1906,17 @@ pub fn get_funded_wallet(
         .set_last_index(KeychainKind::External, funding_address_kix)
         .unwrap();
 
-    let txid = crate::populate_test_db!(wallet.database.borrow_mut(), tx_meta, Some(100));
+    let txid = crate::populate_test_db!(wallet.database.borrow_mut(), tx_meta, Some(100), (@coinbase coinbase));
 
     (wallet, descriptors, txid)
 }
 
+/// Return a fake wallet that appears to be funded for testing.
+pub fn get_funded_wallet(
+    descriptor: &str,
+) -> (Wallet<AnyDatabase>, (String, Option<String>), bitcoin::Txid) {
+    get_funded_wallet_with_coinbase(descriptor, false)
+}
```

Comment:
I couldn't think of any situation where we might need empty inputs in this transaction.. So this function feels redundant..